stories/base/Card.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import Card from "../../components/base/Card";
+import guideIcon from "../../public/images/guides-icon-large.svg";
+
+storiesOf("base|Card", module)
+  .add("default", () => (
+    <Card
+      icon={guideIcon}
+      topic="This is the Topic"
+      description="An existing unverified bank can be verified with microdeposits. Send two small debits and have your Customer verify the amounts."
+    />
+  ))
+  .add("`link`", () => (
```

Comment:
Adding categories to Storybook looks neat! 👍 

I may be missing it, but is there a reason we added back-ticks to the sub-titles like in the following?
>.add("`link`", () => (

Reply:
Ah yeah, I just added the back-ticks to denote that those are prop names.

Reply:
Ah, cool. That sounds good.